Files/Interacts/Interaction.cs

```diff
@@ -599,12 +600,17 @@ public async void DeleteItem_Click(object sender, RoutedEventArgs e)
                 int itemsDeleted = 0;
                 if (selectedItems.Count > 3)
                 {
-                    (App.CurrentInstance as ModernShellPage).UpdateProgressFlyout(InteractionOperationType.DeleteItems, itemsDeleted, selectedItems.Count);
+                    banner = App.CurrentInstance.StatusBarControl.OngoingTasksControl.PostBanner(null, 
+                        CurrentInstance.FilesystemViewModel.WorkingDirectory, 
+                        0, 
+                        UserControls.StatusBanner.StatusBannerSeverity.Ongoing, 
+                        UserControls.StatusBanner.StatusBannerOperation.Delete);
                 }
 
                 foreach (ListedItem storItem in selectedItems)
                 {
-                    if (selectedItems.Count > 3) { (App.CurrentInstance as ModernShellPage).UpdateProgressFlyout(InteractionOperationType.DeleteItems, ++itemsDeleted, selectedItems.Count); }
+                    uint progressValue = (uint)(itemsDeleted * 100.0 / selectedItems.Count);
+                    if (selectedItems.Count > 3) { banner = banner.UpdateProgress(progressValue); }
```

Comment:
I think `IProgress<T>` can be used for normalize progress reporting without implementing a `UpdateProgress()`.
https://docs.microsoft.com/en-us/dotnet/api/system.iprogress-1